app/routes/assets.js

```diff
@@ -1,8 +1,19 @@
+const Joi = require('joi')
+const Boom = require('@hapi/boom')
+
 module.exports = {
   method: 'GET',
   path: '/assets/{path*}',
   options: {
     auth: false,
+    validate: {
+      params: Joi.object({
+        path: Joi.string().required()
+      }),
+      failAction: (_request, _h, _err) => {
+        throw Boom.notFound()
+      }
+    },
```

Comment:
The reason for the error was caused by this route.

If a user attempted to access `/assets` and did not provide a path, then as this is a file directory route, Hapi assumes the users is trying to list all contents of the directory.

For security reasons it blocks that and returns 403.

However, the 403 error prevents the `state` being fully evaluated.  So our cookie plugin set later in journey won't have the `state` object and throws an error.

I've updated the cookie plugin to skip attempting to set the cookie policy on `403` errors as it already does for `404` and `500`.

However, I've also updated this to return a `404` if anyone tries to go direct to the `/assets` path again as there's nothing we want to return from there.